Mathlib/FieldTheory/Minpoly/Basic.lean

```diff
@@ -91,6 +91,11 @@ theorem aeval : aeval x (minpoly A x) = 0 := by
   · exact aeval_zero _
 #align minpoly.aeval minpoly.aeval
 
+/-- Given any `σ : L ≃ₐ[K] L` and any `x : L`, the minimal polynomial of `x` vanishes at `σ x`. -/
+@[simp]
+theorem aeval_algEquiv (f : B ≃ₐ[A] B) (x : B) : (Polynomial.aeval (f x)) (minpoly A x) = 0 := by
```

Comment:
There is no need to assume `f` to be an iso here.

Reply:
Thanks! I have replaced this by `aeval_algHom`.